ko/lessons/libraries/poolboy.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: page
+title: Poolboy
+category: libraries
+order: 2
+lang: ko
+---
+
+동시에 실행 되는 프로세스를 임의로 실행하도록 허용하면 시스템 리소스를 쉽게 소모할 수 있습니다. Poolboy는 워커 풀을 만들어 동시에 실행 되는 프로세스의 수를 제한해 오버헤드가 발생하지 않도록합니다.
+
+## Poolboy를 사용해야하는 이유는 무엇인가요?
+
+잠시 구체적인 예를 생각해 봅시다. 사용자 프로필 정보를 데이터베이스에 저장하기 위한 애플리케이션을 작성한다고 합시다. 모든 사용자 등록에 대해 각각 프로세스를 만들면 무한한 연결 수를 만들게 됩니다. 그리고 어떤 시점에서 이러한 연결은 데이터베이스 서버에서 사용할 수있는 제한된 리소스를 놓고 경합하기 시작합니다. 결국 애플리케이션은 이런 경합으로 발생하는 오버헤드로 인해 시간 초과 및 다양한 예외가 발생하게 됩니다.
+
+이 문제의 해결책은 사용자 등록을 위한 프로세스를 각각 만드는 대신 한정된 연결 수를 사용하는 워커(프로세스)들을 사용하는 것입니다. 그러면 시스템 자원이 모두 소모되는 것을 쉽게 피할 수 있습니다.
+
+여기서 Poolboy가 필요합니다. 수동으로 처리하기 위해 고생할 필요없이 `Supervisor`가 관리하는 워커 풀을 만듭니다. 실제로 많은 라이브러리가 Poolboy를 사용하고 있습니다. 예를 들어 `postgrex`의 연결 풀 *(psql을 사용할 때 Ecto가 사용함)*,`redis_poolex` *(Redis 연결 풀)*가 Poolboy를 사용하는 인기있는 라이브러리중의 일부입니다.
```

Comment:
If you don't mind I want to leave it as is. I will easily forgot this rule, since It is not a concern of mine. Actually, unlike a colon, I think italic text is not that much of a concern nor readability problem. It just look like old fashion for me.

In case of apply it, it will be separated PR, since we need to check the existing translation.